app/assets/stylesheets/library/_font-sizes.scss

```diff
@@ -0,0 +1,5 @@
+$title-size: 2em;
+$header-font-size: 1.5em;
+$homepage-buttons-size: 2.5em;
+$header-table-size: 1.2em;
+$title-size: 2em;
```

Comment:
Can these go in alpha order or is there a specificity issue in doing it?

Reply:
👍